cypress/integration/station29.spec.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * # 「DOM APIを使ってみよう！」 - DOM API
+ */
+
+describe('station29', () => {
+  beforeEach(() => {
+    cy.clock()
+    cy.visit('/station29.html')
+  })
+
+  it('最初はリストで表示されていない', () => {
+    cy.get('#fuits').children('p').should('be.visible')
+  })
+
+  it('ボタンをクリックするとリスト表示される', () => {
+    cy.get('[type="button"]').click()
```

Comment:
```suggestion
    cy.get('[type="button"]').click()
```

buttonでクエリしていない理由は何故ですか？`input[type=button]`を使うのはまれだと思うのですが

Reply:
すみません、調べたのを安易に利用したためです。buttonのクエリで問題ないので修正します。